index.js

```diff
@@ -53,6 +53,14 @@ if (shouldDeleteTags) {
   console.log("🔖  corresponding tags also will be deleted");
 }
 
+let deletePattern = process.env.INPUT_DELETE_TAG_PATTERN;
+if (deletePattern === undefined) {
+  console.log("all releases will be targeted");
+  deletePattern = "";
+} else {
+  console.log(`releases containing ${deletePattern} will be targeted`);
+}
+
```

Comment:
**Nit:**

I just think `console.log("all releases will be targeted");` is redundant. SInce `delete_tag_pattern` is optional, we dont need tell we're targeting all releases. (In my opinion)

I am proposing
```suggestion
let deletePattern = process.env.INPUT_DELETE_TAG_PATTERN || "";
if (deletePattern) {
  console.log(`releases containing ${deletePattern} will be targeted`);
}
```